andela/travel_tool_front
# #161975567 Change title of residence page

#### What does this PR do?
This PR corrects the typographical errors in Residence Page, Documents Page
and restores ellipsis icons on tables

#### Description of Task to be completed?
- update the residence page title to 'RESIDENCE'
- update 'Data Added' to 'Date Added' on Documents Page
- fix broken ellipsis icons on tables
 
#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory -` cd travel_tool_front`
- Pull and checkout into this branch - `ch-change-residence-page-title-161975567`
- Install project dependencies using `yarn install`
- Clone the [Travela backend](https://github.com/andela/travel_tool_back)
- Run database migrations using `yarn run db:rollmigrate`
- Check `.env.example` and set up your `.env` appropriately
- Start the Server and Frontend with `make start` and log into the application
- Create a new travel request
- On the request table, an ellipsis should be visible,
- On Documents Page, a table header should be `Date Added`
- Logged in as a Travel Admin, navigate to `Residence Tab` and Click on `Manage`
  the page should be titled 'Residence' not 'Accommodation'

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#161975567](https://www.pivotaltracker.com/story/show/161975567)


#### Screenshots (if appropriate)
<img width="580" alt="screen shot 2018-11-15 at 5 24 23 pm" src="https://user-images.githubusercontent.com/32720508/48566947-54635a00-e8fc-11e8-8174-b65e3e393ace.png">


#### Questions:
N/A
